ballerina/natives.bal

```diff
@@ -221,103 +221,103 @@ public class Gauge {
     #                     statistics configurations array is passed, then statistics calculation will be disabled.
     #                     If nil () is passed, then default statistics configs will be used for the statitics
     #                     calculation
-    public function init(string name, string? desc = "", map<string>? tags = (),
+    public isolated function init(string name, string? desc = "", map<string>? tags = (),
                StatisticConfig[]? statisticConfig = ()) {
         self.name = name;
         self.description = desc ?: "";
-        self.metricTags = tags ?: DEFAULT_TAGS;
-        self.statisticConfigs = statisticConfig ?: DEFAULT_GAUGE_STATS_CONFIG;
+        self.metricTags = tags.cloneReadOnly() ?: DEFAULT_TAGS;
```

Comment:
I would do 
```suggestion
        self.metricTags = tags is () ? DEFAULT_TAGS : tags.cloneReadOnly();
```

to avoid an unnecessary clone attempt on nil and better readability. Same below.

Reply:
Addressed the review suggestion. Added to another line with the same change.